.github/workflows/release.yaml

```diff
@@ -0,0 +1,91 @@
+---
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::648729606310:role/ecr-maiamcc-contrasite-uploader
+          role-session-name: xwords-upload
```

Comment:
actually hey wait is this supposed to be `contrasite-upload` or is this (name of other website) intentional?